### PR TITLE
[WIP] Sequential exercises

### PR DESCRIPTION
**Pull request by hernanex3**
I will be adding new simple exercises here, and checking the final structure that an exercise should have to be useful for the community.

### CODE REVIEW

File: run_length_encoding/run_length.erl [hunk 1]
```diff
@@ -1,4 +1,25 @@
...
+
+run_length_encode([H|T]) ->
+    run_length_encode([[1, H]|T]);
+
+run_length_encode([]) ->
+    [].
+
+run_length_encoding_test() ->
+    List = [a,a,a,a,b,c,c,a,a,d,e,e,e,e],
+    [[4,a],[1,b],[2,c],[2,a],[1,d],[4,e]] = run_length_encode(List).
```
Comment (hernanex3) [inline run_length_encoding/run_length.erl]: Add spaces between each `,`.

File: run_length_encoding/run_length.erl [hunk 2]
```diff
@@ -1,4 +1,25 @@
 -module(run_length).
--compile(exports_all).
 
-	      
+-export([run_length_encode/1]).
+
+-include_lib("eunit/include/eunit.hrl").
```
Comment (hernanex3) [inline run_length_encoding/run_length.erl]: You may consider writing test in a separate folder `test`.

Comment (facundoolano) [reply run_length_encoding/run_length.erl]: +1 also let's stop using eunit and do all the tests with common test

File: rotatelist/solution/rotations.erl [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    [];
+rotate(L, {Dir, N}) ->
+    case Dir of
+	left ->
+	    {Right, Left} = lists:split(length(L) - N, L);
+	right ->
+	    {Right, Left} = lists:split(N, L)
+    end,
+    lists:append(Left, Right).
+
```
Comment (unbalancedparentheses) [inline rotatelist/solution/rotations.erl]: REMOVE unnecessary lines

File: run_length_encoding/rebar.config [hunk 4]
```diff
@@ -0,0 +1,2 @@
+{erl_opts, [debug_info]}.
+{deps, []}.
```
Comment (unbalancedparentheses) [inline run_length_encoding/rebar.config]: configure your editor to always add a newline at the end of the file. http://adaptivepatchwork.com/2012/03/01/mind-the-end-of-your-line/

**Pull request merged**